packages/web3-eth-transaction-utils/src/utils/get_transaction_gas_pricing.ts

```diff
@@ -90,7 +125,11 @@ export async function getTransactionGasPricing<ReturnFormat extends DataFormat>(
 			(transactionType === '0x0' || transactionType === '0x1')
 		)
 			return {
-				gasPrice: await getGasPrice(web3Context, returnFormat),
+				gasPrice: format(
+					{ eth: 'uint ' },
+					await ethRpcMethods.getGasPrice(web3Context.requestManager),
+					returnFormat,
+				) as FormatType<Numbers, ReturnFormat>,
```

Comment:
`getGasPrice` from `web3-eth` was formatting the value for us, but as mentioned above, we now have to use the simple `rpc_method_wrapper` from `web3-rpc-methods` to avoid circular dependencies